duckduckgo/iOS
# Add Back/Forward navigation long press menu

Task/Issue URL: https://app.asana.com/0/414235014887631/1202297990501898/f

**Description**:
This PR adds a feature for users to long press the back/forward buttons on the navigation bar and be presented with a menu of options that the user can go directly to.

**Steps to test this PR**:
1. Visit a bunch of different websites
2. Long press the back button and check if the sites you visited are there
3. Select a website in the middle, check if the site loaded correctly
4. Long press the forward button and check if the stack displays the websites you visited after the current one.
____
1. Check if the order of the websites on the back button are newest at the bottom, oldest at the top
2. Check if the order of the websites on the forward button are newest at the top oldest at the bottom
----- 
1. Check if protections are being respected after reloading a website from the back button
----- 
1. This feature is iOS 14+ only, check if this is being respected.
----- 

**Device Testing**:


**OS Testing**:


**Theme Testing**:
